src/app/api/_api-services/onchain_db_service/subsquid_service/index.ts

```
@@ -1057,4 +1064,276 @@ export class SubsquidService extends SubsquidUtils {
+			categoryCounts: {
+				governance: groupResults.Governance ?? null,
+				main: groupResults.Main ?? null,
+				treasury: groupResults.Treasury ?? null,
+				whiteList: groupResults.Whitelist ?? null
+			}
+		};
+	}
+
+	static async GetTurnoutPercentageData({ network }: { network: ENetwork }): Promise<ITurnoutPercentageData> {
+		try {
+			const { data: subsquidData, error: subsquidErr } = await this.subsquidGqlClient(network).query(SubsquidQueries.GET_TURNOUT_PERCENTAGE_DATA, {}).toPromise();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)